gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1162,7 +1171,7 @@ function CheckForMapSwitch()
 
 	if switchmap then
 		timer.Stop("end2prep")
-		timer.Simple(15, game.LoadNextMap)
+		hook.Run("TTT2LoadNextMap")
```

Comment:
you should add `nextmap` as param and an empty line in front of this line to seperate hooks from other code blocks :)

Reply:
Done